src/reconstruct/reconstruct.ts

```diff
@@ -57,10 +71,17 @@ const reconstructAsLeaf = (leaf: RNodeWithParent, configuration: ReconstructionC
 
 const foldToConst = (n: RNodeWithParent): Code => plain(getLexeme(n))
 
+//look up exact function
+/*
+--helper function--
+*/
 function indentBy(lines: Code, indent: number): Code {
 	return lines.map(({ line, indent: i }) => ({ line, indent: i + indent }))
 }
 
+/*
+--recunstruct--
```

Comment:
is this continued `reconstruct`-typo intended?

Reply:
the typo is not intended
i will fix this in the next commit